server/registry/internal/storage/list.go

```diff
@@ -998,42 +998,3 @@ func artifactMap(artifact models.Artifact) (map[string]interface{}, error) {
 		"size_bytes":  artifact.SizeInBytes,
 	}, nil
 }
-
```

Comment:
It's unfortunate that we now don't have any way to list tags (other than raw database queries). Nonetheless, I think this is the right change to make now, and it's really helpful to have this in one commit in case we want to quickly find this code to revive it.